### PR TITLE
add Impl for llm edge benchmark suite

### PR DESCRIPTION
**Pull request by XueSongTap**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

add Impl for llm edge benchmark suite

**Which issue(s) this PR fixes**:

https://github.com/kubeedge/ianvs/pull/127

### CODE REVIEW

**Review by MooreZheng — CHANGES_REQUESTED** [review]
There are CI issues that remain to be resolved, see https://github.com/kubeedge/ianvs/actions/runs/11179368172?pr=150

**Review by MooreZheng — CHANGES_REQUESTED** [review]
The CI issues are properly tackled. As discussed at the routine meeting, suggestions would be 
1. Might show a final result (e.g., ranking) in the PR;
2. Replace the Chinese statements into English; 
3. Replace the print with loggers;

File: examples/llm-edge-benchmark-suite/single_task_bench/testenv/prefill_latency.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from sedna.common.class_factory import ClassType, ClassFactory
+
+__all__ = ["prefill_latency"]
+
+@ClassFactory.register(ClassType.GENERAL, alias="prefill_latency")
+def prefill_latency(y_true, y_pred):
+    # avg_prefill_latency = y_pred.get('avg_prefill_latency', [])
+    # return avg_prefill_latency
+    #TODO 前面所有歌predict_dict 的结果，可以通过下面拿出来，我想把计算平均的过程放在这里，帮我实现
```
Comment (hsj576) [inline examples/llm-edge-benchmark-suite/single_task_bench/testenv/prefill_latency.py:9]: The Chinese prompt should be removed.

**Review by hsj576 — CHANGES_REQUESTED** [review]
The Chinese prompt should be removed.

**Comment by XueSongTap** [comment]
@hsj576 Could you please review this PR? I've made several documentation and configuration updates:

- Comment cleanup in basemodel.py
- Configuration parameter adjustments
- Documentation improvements in README, including remove chinese part

Here's  screenshot of the benchmark ：
image
image




The results show the changes are working as expected,

Let me know if you need any additional information or clarification.

**Review by MooreZheng — COMMENTED** [review]
This pull request contains 10 commits, which might make maintenance difficult, considering the number of contributors, pull requests, and their commits in KubeEdge Ianvs recently. In the final stage, @XueSongTap  can squash the commits into one using [rebase techniques](https://blog.kinboy.wang/2019/04/09/Git-%E5%8E%8B%E7%BC%A9%E5%A4%9A%E4%B8%AAcommit%E4%B8%BA%E5%8D%95%E4%B8%AAcommit/).

**Comment by XueSongTap** [comment]
@MooreZheng I have already addressed this concern by squashing all commits into a single commit using rebase. The changes are now consolidated into one cohesive commit for better maintenance and clearer history. Please review the updated changes.

**Comment by MooreZheng** [comment]
@XueSongTap might need one more review from @hsj576

**Review by MooreZheng — CHANGES_REQUESTED** [review]
To avoid conflicts, need to change print() using logger

**Comment by XueSongTap** [comment]
@MooreZheng hi, i've changed print() using logger. Could you please review this PR? Let me know if you need any additional information or clarification.

**Review by MooreZheng — APPROVED** [review]
All concerns are fixed. Well done and congratulations to @XueSongTap

**Pull request merged**